src/sentry/auth/access.py

```diff
@@ -936,6 +938,7 @@ def from_request_org_and_scopes(
     scopes: Iterable[str] | None = None,
 ) -> Access:
     is_superuser = is_active_superuser(request)
+    breakpoint()
```

Comment:
Project just leftover from debugging!

Reply:
yes it's leftover in case anybody wants to see what i'm seeing :'(